src/pacmod3_core.cpp

```diff
@@ -43,7 +46,9 @@ const uint32_t AS::Drivers::PACMod3::ParkingBrakeRptMsg::CAN_ID = 0x224;
 const uint32_t AS::Drivers::PACMod3::ShiftRptMsg::CAN_ID = 0x228;
 const uint32_t AS::Drivers::PACMod3::SteerRptMsg::CAN_ID = 0x22C;
 const uint32_t AS::Drivers::PACMod3::TurnSignalRptMsg::CAN_ID = 0x230;
-const uint32_t AS::Drivers::PACMod3::WiperRptMsg::CAN_ID = 0x234;
+const uint32_t AS::Drivers::PACMod3::EngineBrakeRptMsg::CAN_ID = 0x244;
+const uint32_t AS::Drivers::PACMod3::MarkerLampRptMsg::CAN_ID = 0x248;
+const uint32_t AS::Drivers::PACMod3::SprayerRptMsg::CAN_ID = 0x238;
```

Comment:
Same here with sorting. Also, it looks like WiperRptMsg was accidentally removed.